components/Onboarding/ConnectViaWallet/ConnectViaWallet.store.tsx

```diff
@@ -5,7 +5,6 @@ import { createStore, useStore } from "zustand";
 type IConnectViaWalletStoreProps = {
   address: string;
   alreadyV3Db: boolean;
-  onXmtp: boolean;
```

Comment:
can you help me understand why you're removing this?

Reply:
It's not used any more

There used to be a possible 3 different signatures

2 for V2 (1 was only done once ever, which is why this check existed in the first place)
1 for V3

Now since there is a V3 only we only need one signature ever

Reply:
thank you for explaining